apps/sms/views/inbox/js/startup.js

```diff
@@ -39,6 +40,7 @@
       ConversationClient.init(App.instanceId);
       MessageManager.init();
       Navigation.init();
+      Drafts.init();
```

Comment:
I might not add this right now since we don't handle draft change in conversation view, but It's interesting question about handling the draft change in conversation. If we open the same conversation view in another instance, should we sync the draft status as well? It looks like the instance opened in another instance will also load the saved draft correctly, so maybe it's a good idea to sync the changes to original instance.